bigtable-hbase-1.x-parent/bigtable-hbase-1.x-benchmarks/pom.xml

```diff
@@ -9,23 +9,6 @@
 
   <artifactId>bigtable-hbase-1.x-benchmarks</artifactId>
 
-  <!--  TODO: remove this dependency when upgraded transitively to 1.34.2 -->
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>com.google.http-client</groupId>
-        <artifactId>google-http-client-jackson2</artifactId>
-        <version>1.35.0</version>
-      </dependency>
-      <!--  todo(kolea2): temporary workaround until api-common and google-cloud-core are updated to 1.7.1-->
```

Comment:
@kolea2 https://search.maven.org/artifact/com.google.cloud/google-cloud-shared-dependencies/0.8.3/pom has api-common 1.9.3 and google-cloud-core 1.93.7.